index.html

```diff
@@ -6,9 +6,37 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>portfolio</title>
 
+    <!-- Bootstrap icons link -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css"
+    />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1 class="header">"Hello!"</h1>
+    <header class="header">
+      <nav class="navbar">
+        <div class="logo">BZ.</div>
+        <i class="bi bi-list"></i>
+      </nav>
+
+      <section class="hero">
+        <div class="hero__title">
+          <h1>
+            I'm Bahoz<br />
+            Glad to see<br />
+            you!
+          </h1>
+        </div>
+        <p class="hero__text">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and <br />
+          experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>

```

Comment:
- Please make this section text have a font size of `16px` as in the Figma design, yours is currently `15px`, as this will make you learn how to stick to details more in future jobs.
##### Yours
![Screenshot from 2022-05-25 22-04-12](https://user-images.githubusercontent.com/58771507/170367056-e4cbacf4-2418-45de-bd14-7729a4cd8c60.png)